### ISSUE TITLE
AV01 format and vlc 

### ISSUE THREAD

**Issue by sprajagopal** [omega-gg/MotionBox#20]
Thanks for this wonderful project! I think this could serve as the primary method of watching videos.
I can see that some videos just don't play because of the newer AV01 format (which vlc refuses to incorporate into their player for various reasons). This could be one of the main issues of using this as a primary driver.
There seem to be other players in Linux that can play AV format. Is there a way to use other players with Motionbox?

**Comment by 3unjee** [comment]
Yeah I'll look into that, by default the Youtube backend tends to return AV01 video, which is a bit bold.
